pnpm-workspace.yaml

```diff
@@ -4,5 +4,7 @@ packages:
     - 'dapps/**'
     - '!**/dist/**'
     - '!sdk/typescript/builder'
+    - '!sdk/typescript/client'
+    - '!sdk/typescript/cryptography'
```

Comment:
Can we lint for this or something?

Reply:
maybe?

`pnpm build` currently generates the package.json files, it could also ensure it gets added here, if that works for you

Reply:
Yeah maybe that's a decent solution for now at least. I'm increasingly feeling like we should just generate an output package directory entirely though

Reply:
And I assume adding `'!sdk/typescript/*'` doesn't work?

Reply:
Yeah, I think eventually that's probably going to be better.  This is primarily because of an inconsistency with manypkg (a linter we use for package.json files) and it only happens in CI

Reply:
I'm fine tearing out manypkg too if it causes friction. It was mostly just so that we didn't end up with a bunch of duplicate versions.